tests/test_bounce_rate.py

```diff
@@ -93,9 +93,9 @@ def test_set_total_notifications(self, mocked_bounce_rate_client, mocked_service
         [
             (10, 100, 0.1),
             (5, 100, 0.05),
-            (5, 1000, 0.01),  # inexact b/c we are rounding to 2 decimal places
-            (5, 10000, 0.0),  # inexact b/c we are rounding to 2 decimal places
-            (5, 100000, 0.0),  # inexact b/c we are rounding to 2 decimal places
+            (5, 1000, 0.005),  # inexact b/c we are rounding to 2 decimal places
+            (5, 10000, 0.0005),  # inexact b/c we are rounding to 2 decimal places
+            (5, 100000, 0.00005),  # inexact b/c we are rounding to 2 decimal places
```

Comment:
You could remove the comments from these lines - they are now out of date since we are not rounding.